Packages.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup Label="Dependency Versions">
-    <_CluedIn>4.0.0-*</_CluedIn>
+    <_CluedIn>4.1.0-alpha.*</_CluedIn>
```

Comment:
/release/4.0.0 is producing 4.0.0-beta
https://dev.azure.com/CluedIn-io/CluedIn/_build/results?buildId=74089&view=logs&j=57d2438d-e7eb-5a97-bac8-213dcbc3faf0&t=3078d362-10fa-545b-7ce8-eea4b3470f2e